test/e2e/certificates_test.go

```diff
@@ -171,10 +185,109 @@ var _ = Describe("ACME Certificate", Ordered, func() {
 			err = verifyCertificate(ctx, certName, ns.Name, certDomain)
 			Expect(err).NotTo(HaveOccurred())
 		})
+
+		It("should obtain a valid LetsEncrypt certificate on ibm cloud CIS", func() {
+			cisCRN, isCisCRN := os.LookupEnv(cisCRNEnvironmentVar)
+			if targetPlatform, ok := os.LookupEnv(targetPlatformEnvironmentVar); ok && targetPlatform == "ibmcloud" {
```

Comment:
Are these values setup in IBM Cloud OpenShift clusters by default, or you're running them in a specific environment where such environment variables are available?

Reply:
We're running them in a specific environment where such environment variables are passed in . For example:

`TARGET_PLATFROM=ibmcloud CIS_CRN make test-e2e`

Reply:
In case that is IBM Cloud on UPI, can we make it something that can take the UPI keyword into consideration. How about "ibmcloud-upi" instead? That'd work for us for the mid term (and also helps not conflict if we have ibmcloud-ipi in the future).

Reply:
Sure - @pranitaT can you switch the var to ibmcloud-upi